pages/docs/specifications/v2.0.0.md

```diff
@@ -1,3 +1,5 @@
+# AsyncAPI Specification
+
```

Comment:
Now that the heading does not break the TOC, should we remove the automation that removes the heading?

Reply:
yup, less custom actions on the spec is better